src/components/drawer/drawer.module.scss

```diff
@@ -5,7 +5,7 @@
     top: 0px;
     right: 0px;
     bottom: 0;
-    background-color: rgba(148, 148, 148, 0.5);
+    background: rgba(148, 148, 148, 0.5);
```

Comment:
should this be moved to constants?

Reply:
I decided not to add colors with alpha channels to variables...